build-tools/create-packs/Microsoft.Android.Runtime.proj

```diff
@@ -11,7 +11,7 @@ projects that use the Microsoft.Android framework in .NET 5.
 
   <PropertyGroup>
     <AndroidRID Condition=" '$(AndroidRID)' == '' ">android.21-arm64</AndroidRID>
-    <AndroidABI Condition=" '$(AndroidABI)' == '' ">arm64-v8a</AndroidABI>
+    <AndroidABI Condition=" '$(AndroidABI)' == '' ">arm64-v8a-ne6</AndroidABI>
```

Comment:
Should this be `-net6`, not `-ne6`?  `-net6` is consistent with the changes in `Directory.Build.props`.